src/Microsoft.AspNetCore.Mvc.Testing/WebApplicationTestFixture.cs

```diff
@@ -34,7 +44,6 @@ public class WebApplicationTestFixture<TStartup> : IDisposable where TStartup :
         /// </para>
         /// </summary>
         public WebApplicationTestFixture()
```

Comment:
I would love to kill all the other constructors/static methods except for this one. If someone wants to setup the content root manually they can do so by overriding ConfigureWebHostBuilder. Thoughts @rynowak?

Reply:
I'm not convinced that's a good thing